docs/components/base/_index.md

```diff
@@ -208,7 +208,7 @@ Set the linear and angular power of the base, represented as a percentage of max
 
 - `linear` [(Vector3)](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Vector3): The percentage of max power of the base's linear propulsion.
   In the range of -1.0 to 1.0, with 1.0 meaning 100% power.
-  Viam's coordinate system considers +Y to be the forward axis (+/- X left/right, +/- Z up/down), so use the Y component of this vector to move forward and backward when controlling a wheeled base.
+  Viam's coordinate system considers +Y to be the forward axis (+/- X right/left, +/- Z up/down), so use the Y component of this vector to move forward and backward when controlling a wheeled base.
```

Comment:
Yeah, we have [a ticket](https://viam.atlassian.net/jira/software/c/projects/DOCS/boards/31?modal=detail&selectedIssue=DOCS-845&quickFilter=41) around this up right now but I think it's not super well understood internally? I know that was a suggestion/reviewed at some point. If you have confirmation that this is correct that'd be great and it would help us out on that other ticket as well!

Reply:
Which ticket is it? The link points just to the Docs board. 

I am confident this is correct, though I added @raybjork to help me sanity-check the suggested changes.